pom.xml

```diff
@@ -67,6 +68,27 @@
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.github.searls</groupId>
+        <artifactId>jasmine-maven-plugin</artifactId>
+        <version>1.3.1.6</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>test</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <jsSrcDir>${project.basedir}/src/main/resources/public/js</jsSrcDir>
```

Comment:
This doesn't seem to be the right path.

It doesn't seem to be breaking anything, but can we update it to `${project.basedir}/src/main/webapp`?

Reply:
For some incomprehensible reason, this causes an error.

Reply:
main/resources/public/js doesn't exist in our repository... What was the error you are running into, can you be more specific?